src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/Arguments/LaunchTimeoutArgument.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.DotNet.XHarness.Common.CLI.CommandArguments;
+
+namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Android
+{
+    /// <summary>
+    /// Time to wait for boot completion. Defaults to 5 minutes.
```

Comment:
I think it's not correct to mention here default value since it's not set up inside this class